plugins/single_plugins/alpha.cpp

```diff
@@ -73,6 +74,22 @@ class wayfire_alpha : public wf::plugin_interface_t
         return wf::ipc::json_ok();
     };
 
+    wf::ipc::method_callback ipc_get_view_alpha = [=] (nlohmann::json data) -> nlohmann::json
+    {
+        WFJSON_EXPECT_FIELD(data, "view-id", number_unsigned);
+
+        auto view = wf::ipc::find_view_by_id(data["view-id"]);
+        if (view && view->is_mapped())
+        {
+            auto response = wf::ipc::json_ok();
+            response["alpha"] = ensure_transformer(view)->alpha;
```

Comment:
ensure_transformer will add a new transformer to the view even if the alpha==1.0. It would be better to check whether a transformer exists (if not, alpha == 1.0) and if it does, then get its alpha.

Reply:
done